src/Symfony/Bundle/FrameworkBundle/Tests/CacheWarmer/TemplateFinderTest.php

```diff
@@ -50,11 +50,11 @@ function ($template) { return $template->getLogicalName(); },
         );
 
         $this->assertCount(7, $templates, '->findAllTemplates() find all templates in the bundles and global folders');
-        $this->assertContains('BaseBundle::base.format.engine', $templates);
-        $this->assertContains('BaseBundle::this.is.a.template.format.engine', $templates);
-        $this->assertContains('BaseBundle:controller:base.format.engine', $templates);
-        $this->assertContains('BaseBundle:controller:custom.format.engine', $templates);
-        $this->assertContains('::this.is.a.template.format.engine', $templates);
-        $this->assertContains('::resource.format.engine', $templates);
+        $this->assertStringContainsString('BaseBundle::base.format.engine', $templates);
```

Comment:
`templates` is an array, you should revert and keep `assertContains`.

The deprecation is about using `assertContains` when the second argument is a string (ie. `assertContains('foo', 'foobarbaz')` )
Same comment for many changes bellow. 

An easy way to spot theme, is to run test and revert reverting the failing tests ;-)